### DISCUSSION TITLE
New 3060 manufactured from may

### DISCUSSION THREAD

**Discussion by maickelbr** [Lolliedieb/lolMiner-releases#654] [General]
Does lolminer 1.30 unlock the new 3060s manufactured from May onwards?

**Comment by japex-brz** [comment]
Hi...
I guess the LolMiner unlocker wont work with the newer RTX3060 (GA-106-**_302_**)
At least it doesnt worked for me. 
As it asks to downgrade to a previous driver 460.39 wich is from Jan/2021, it seems that HiveOS wont find the card so im getting black screen after reboot. 
I guess its does not come named with LHR but it works equal.
On windows it wont work either because 407.05 developer driver also wont find the card. 
So im stuck with 15MHs :-(

**Reply by 5helloworld** [reply]
I bought an Leadtek-NVIDIA Public RTX3060 LHR (shipped by the seller by mistake), the manufacturing date is the 22nd week of 2021. Yesterday I used the latest version of hiveos and NVIDIA driver 465.31 to successfully drive the graphics card before going to bed. With the help of the latest version of lolminer, a stable eth hash rate of 32.5mh/s has been achieved, and it has worked more than 24. But I am puzzled that it works at 16mh/s first. After a few hours, I woke up and found that it was working at 32.5mh/s.
![image](https://user-images.githubusercontent.com/24307179/126493599-68472b27-187c-4923-ac76-18113059ee90.png)
My use parameters are as follows
![image](https://user-images.githubusercontent.com/24307179/126493029-a464e299-98a2-41db-bf13-e12a55d80a77.png)
Before the hiveos and NVIDIA drivers are updated, the graphics card is recognized as device 2504
![image](https://user-images.githubusercontent.com/24307179/126494694-09f16332-3e63-425a-9c0f-71a0da4971d2.png)
Recognized as RTX 3060 LHR after update
![image](https://user-images.githubusercontent.com/24307179/126493259-2d1950b6-4f84-4028-8dec-d2ee5dfe11ed.png)
